### PR TITLE
test(system-e2e): More tests

### PR DESCRIPTION
**Pull request by krrrrrrrrrrr**
## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -226,6 +226,7 @@
     "pdfkit": "0.11.0",
     "pg": "8.1.0",
     "pg-hstore": "2.3.3",
+    "playwright": "^1.31.2",
```
Comment (eirikurn) [inline package.json:229]: Fixed dep?

**Review by eirikurn — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (eirikurn) [reply package.json:229]: I mean the dependency should not have a "^".

Though do we need this dependency? Or just upgrade @playwright/test? I notice the playwright docs don't install it manually.

File: scripts/ci/Dockerfile [hunk 2]
```diff
@@ -134,6 +134,7 @@ RUN apt-get -y install awscli zip
 
 COPY --chown=pwuser:pwuser ${APP_HOME}/package.json .
 RUN yarn install
+RUN yarn playwright install
```
[the remark this answers is not in the document]
Comment (krrrrrrrrrrr) [reply scripts/ci/Dockerfile:137]: This is `playwright` installing its dependencies. The version of `playwright` running the tests is defined in the base image `mcr.microsoft.com/playwright:v1.31.2-focal` (i.e. version `1.31.2`)

**Review by peturgq — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (krrrrrrrrrrr) [reply package.json]: I set the version with the playwright image we are using. Good catch :)

**Pull request merged**